.github/workflows/test_release_unix_nightly.yml

```diff
@@ -0,0 +1,36 @@
+name: Test latest release nightly Ubuntu
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # nightly:
+    - cron:  '0 0 * * *'
+
+jobs:
+  build:
+
+    strategy:
+      max-parallel: 4
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7]
+
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Clone and install dependencies
+      run: |
+        git clone https://github.com/awslabs/gluon-ts --branch $(curl https://api.github.com/repos/awslabs/gluon-ts/releases/latest | grep tag_name | cut -d : -f 2,3 | tr -d \"\ | tr -d \,\ )
+        cd gluon-ts
+        python -m pip install --upgrade pip
+        pip install -r requirements/requirements-mxnet.txt
+        pip install gluonts
```

Comment:
I think this will install gluonts from pypi? And that's totally fine, but probably the logic for getting the latest tag is superfluous then

Reply:
I see, that's needed to get the right dependencies

Reply:
There might be an issue here in case we ever start doing release branches, and releasing bug fixes for previous releases: the latest tag might not be the same as the latest available release from PyPI. But until then, this is fine.

Reply:
yep, also if we add more requirement files that the tests depend on as is done [here](https://github.com/awslabs/gluon-ts/blob/53120d4af666e24fe92a2f5051f865399ad7a7da/.github/workflows/tests.yml#L29) we will run into issues for future releases